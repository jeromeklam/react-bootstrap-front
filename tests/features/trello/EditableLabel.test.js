import React from 'react';
import renderer from 'react-test-renderer';
import EditableLabel from '../../../src/features/trello/EditableLabel';

const callback = jest.fn();
jest.useFakeTimers();

it('render with className and name', () => {
  // First render
  const component = renderer.create(
    <EditableLabel className="btn bg-secondary" value="TEST"/>
  );
  let test = component.toJSON();
  expect(test).toMatchSnapshot();
});
