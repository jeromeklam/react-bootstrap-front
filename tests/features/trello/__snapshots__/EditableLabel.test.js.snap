// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render with className and name 1`] = `
<div
  className="comPlainTextContentEditable "
  contentEditable="true"
  onBlur={[Function]}
  onInput={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onPaste={[Function]}
  placeholder={false}
/>
`;
